docs/swagger2.yml

```diff
@@ -189,28 +203,28 @@ paths:
             $ref: '#/definitions/Error'
     get:
       operationId: "GetApp"
-      summary: "Get information for an app."
-      description: "Get information for an app."
+      summary: "Get app metadata"
+      description: "Get metadata for an Application."
       tags:
         - Apps
       parameters:
-        - $ref: '#/definitions/appPathParam'
+        - $ref: '#/definitions/appIDParam'
       responses:
         200:
-          description: App details.
+          description: App metadata
           schema:
             $ref: '#/definitions/AppWrapper'
         404:
-          description: App does not exist.
+          description: App does not exist
           schema:
             $ref: '#/definitions/Error'
         default:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-    patch:
+    post:
       operationId: "UpdateApp"
-      summary: "Updates an app."
+      summary: "Updates an app"
```

Comment:
we do want to remove patch, was going forward with only having GET/PUT/DELETE for each resource, would be nice to have idempotent create or update (post doesn't necessarily suggest such semantics)